### PR TITLE
crip: init at 3.9; perlPackages.CDDB_get: init at 2.28

### PR DESCRIPTION
**Pull request by endgame**
###### Motivation for this change

I want to rip my music CDs on my NixOS box.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [X] other Linux distributions (ubuntu 16.04.4, sandbox = true)
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [?] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Notes:

* No maintainers are set - I'm happy to maintain these if you want; they're finished packages, really. If so, I'll add fields and rebase.

### CODE REVIEW

File: pkgs/applications/audio/crip/default.nix [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        --set PERL5LIB "${makePerlPath [ perlPackages.CDDB_get ]}" \
+        --set PATH "${toolDeps}"
+    done
+  '';
+
+  meta = {
+    homepage = http://bach.dynet.com/crip/;
+    description = "Terminal-based ripper/encoder/tagger tool for creating Ogg Vorbis/FLAC files";
+    license = stdenv.lib.licenses.gpl1;
+    platforms = stdenv.lib.platforms.linux;
```
Comment (Mic92) [inline pkgs/applications/audio/crip/default.nix]: Do you want to maintain this package?

Comment (endgame) [reply pkgs/applications/audio/crip/default.nix]: Yeah ok.

Comment (Mic92) [reply pkgs/applications/audio/crip/default.nix]: the process is described here: https://nixos.wiki/wiki/Nixpkgs#Becoming_a_Nixpkgs_maintainer

**Comment by endgame** [comment]
I've added myself to the maintainer list and added myself to the packages, rebased and pushed.

**Pull request merged**